_posts/2023-04-05-#27255.md

```diff
@@ -10,14 +10,44 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+### Background and prior work
+
+_Miniscript_ is a language for writing (a subset of) Bitcoin Scripts in a structured way, enabling analysis, composition, generic signing and more. _Miniscript_ support in Bitcoin Core has been an ongoing effort, with watch-only support for miniscript descriptors added in `v24.0.1`. For background:
+
+* [https://bitcoin.sipa.be/miniscript](https://bitcoin.sipa.be/miniscript) gives an overview of the miniscript langauge
+* [#24147](https://github.com/bitcoin/bitcoin/pull/24147) provides the rationale and initiale backbone needed for miniscript support in Bitcoin Core
+* [#24148](https://github.com/bitcoin/bitcoin/pull/24148) adds watch-only support for miniscript descriptors
+* [#24149](https://github.com/bitcoin/bitcoin/pull/24149) adds signing support for miniscript descriptors
+
+### TapMiniscript
+
+While [_Tapscript_](https://github.com/bitcoin/bips/blob/master/bip-0342.mediawiki) shares most operations with legacy Bitcoin Script, there are a few notable differences. As such, _TapMiniscript_ has been a project to support tapscript in miniscript. For background:
+
+* [This gist](https://gist.github.com/sipa/06c5c844df155d4e5044c2c8cac9c05e) contains the initial design and discussion on supporting miniscript in tapscript
+* [#134](https://github.com/sipa/miniscript/pull/134/) adds tapscript support in miniscript
+
+### Putting it all together
+
+Putting it all together, this PR adds support for Tapscript in Miniscript in Bitcoin Core and makes Miniscript available inside `tr()` descriptors, both with watching and signing support.
+
 ## Questions
+
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
+1. In your own words, what are some of the differences between Segwit v0 Script (Bitcoin Script with the Segwit Soft Fork rules) and Segwit v1 script (Tapscript)?
+1. What is Miniscript?
+1. What is Tapscript?
+1. This PR [adds](https://github.com/bitcoin/bitcoin/pull/27255/commits/c0ba8ebbf6369b37b645165bb5cd638fc7eee67f) a `multi_a` fragment for tapscript. How is this different than the existing `multi` fragment? Does it have the same properties as `multi`?
+1. In miniscript, we have type modifiers, which guarantee addititional properties for an expression. [866284d](https://github.com/bitcoin/bitcoin/pull/27255/commits/866284d007993551f681809d9e48175a3b0fe0c1) makes "**d**" have the "**u**" property under Tapscript.
+	* What are the "**d**" and "**u**" type modifiers?
+	* Why is it that we can make **d** have the **u** property here? Why not in non-tapscript miniscript?
+1. The majority of complexity in this PR comes from statically ensuring no spending path exceeds the stack size at execution time:
```

Comment:
nit: i know it's my words but could we get rid of "complexity"? I'm not sure it's really that complex, and there are other things that may seem complicated, idk.
```suggestion
1. This PR adds some logic for statically ensuring no spending path exceeds the stack size at execution time:
```